packages/share_plus/share_plus/ios/Classes/FPPSharePlusPlugin.m

```diff
@@ -358,6 +358,8 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
                     atSource:originRect
                     toResult:result
                   withResult:withResult];
+          if (!withResult)
+            result(nil);
```

Comment:
`withResult` may always be false in shareUri, so I am tempted to just always call to `result(nil)`.

However, since this value comes from the Dart side, we cannot warranty that in native code. That's why I added the if-condition here.